clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ClusterController.groovy

```diff
@@ -43,6 +44,7 @@ class ClusterController {
   @Autowired
   MessageSource messageSource
 
+  @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ')")
   @RequestMapping(method = RequestMethod.GET)
   Map<String, Set<String>> list(@PathVariable String application) {
```

Comment:
It gives cluster names grouped by account in the scope of an application.

It seems that the annotations don't work well with Maps, since it's not a Collection. I see the call route through Gate, but don't see anywhere (in a cursory look) in Deck for where it's used. Perhaps @anotherchrisberry or @danielpeach might know?